draft-ietf-taps-interface.md

```diff
@@ -318,8 +321,8 @@ If none of the available transport protocols provides Preservation of Message
 Boundaries, but there is a transport protocol that provides a reliable ordered
 byte stream, an application could receive this byte stream as partial
 Messages and transform it into application-layer Messages.  Alternatively,
-an application might provide a Message Framer, which can transform a
-byte stream into a sequence of Messages ({{framing}}).
+an application might provide a Message Framer, which transforms a
+byte stream to or from a sequence of Messages ({{framing}}).
```

Comment:
```suggestion
sequence of Messages into a byte stream and vice versa ({{framing}}).
```

I like your suggestion better than what we have (the original just maps better to why this is really necessary in our own minds IMO), but now it seems awkward to begin with the byte stream... I just think that it reads a bit nicer with the change above.